spec/models/manageiq/providers/openstack/cloud_manager/vm_spec.rb

```diff
@@ -58,7 +58,7 @@
       end
 
       it "checks evacuation is_available?" do
```

Comment:
Same here (If this is a pattern you may want to review all of the previous PRs and update the description accordingly)

I'll just leave this here https://twitter.com/nzkoz/status/538892801941848064  :)